geo_lookup/geo_mag_declination.cpp

```diff
@@ -101,17 +101,28 @@ static constexpr float get_table_data(float lat, float lon, const int8_t table[L
 	return lat_scale * (data_max - data_min) + data_min;
 }
 
-float get_mag_declination(float lat, float lon)
-{
-	return get_table_data(lat, lon, declination_table);
+float get_mag_declination_radians(float lat, float lon) {
```

Comment:
```suggestion
float getMagDeclinationRadians(float lat, float lon) {
```

Reply:
When we already update the interface, why not also enforce camelCase

Reply:
> why not also enforce camelCase

Lack of any style guide or enforcement tool. I don't want to go crazy adopting an enormous style guide, but clearly we need something minimal in place to get the naming consistent.

Reply:
> Lack of any style guide or enforcement tool. 

I'll look at getting something simple together.